hazelcast/src/main/java/com/hazelcast/map/impl/MapStoreWrapper.java

```diff
@@ -99,7 +100,8 @@ public void deleteAll(Collection keys) {
 
     public Iterable loadAllKeys() {
         if (isMapLoader()) {
-            return mapLoader.loadAllKeys();
+            // Invoke reflectively to preserve backwards binary compatibility
+            return ReflectionHelper.invokeMethod(mapLoader, "loadAllKeys");
```

Comment:
On the other hand we set things accessible in many other places so can't see it making a difference